imports/client/components/JoinHunt.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import acceptHuntInvitationCode from "../../methods/acceptHuntInvitationCode";
+
+const JoinHunt = () => {
+  const invitationCode = useParams<"invitationCode">().invitationCode!;
+  const [status, setStatus] = useState<string>("loading...");
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    acceptHuntInvitationCode.call({ invitationCode }, (error, huntId) => {
+      if (error) {
+        setStatus(error.reason ?? "Unknown error");
+      } else {
+        navigate(`/hunts/${huntId}`);
+      }
+    });
+  }, [invitationCode, navigate]);
```

Comment:
@zarvox Since you did the `StrictMode` migration work, is this going to result in calling the method more than once?

Reply:
AIUI - yes, in strict mode, this method will be invoked twice ([ref](https://react.dev/reference/react/useEffect#caveats)). But it is more or less idempotent - if the user is already a member of the given hunt, it will log a message (at info) but otherwise return successfully. LMK if I should handle this in some other way though.